frontend/sozisel-app/src/components/TemplateCreation/EventsList/EventList.tsx

```diff
@@ -1,29 +1,55 @@
 import "./EventsList.scss";
 
-import React, { ReactElement } from "react";
+import React, { ReactElement, useCallback, useMemo } from "react";
+import { useDeletePollMutation, useDeleteQuizMutation } from "../../../graphql";
 
 import { Event } from "../../../model/Template";
+import EventListElement from "./EventsListElement/EventListElement";
 import { Paper } from "@material-ui/core";
-import Quiz from "./EventsListElement/Quiz/QuizEvent";
 
 interface EventListProps {
   events?: Event[];
 }
 
 export default function EventList({ events }: EventListProps): ReactElement {
+  const [deleteQuizMutation] = useDeleteQuizMutation({
+    refetchQueries: ["SessionTemplate"],
+  });
+  const [deletePollMutation] = useDeletePollMutation({
+    refetchQueries: ["SessionTemplate"],
+  });
+
+  const sortedEvents = useMemo(() => {
+    if (events) {
+      return [...events].sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
Shouldn't we sort events by theirs `startMinute` property

Reply:
I don't know, its not a part of this PR.

Reply:
I'm not the author.

Reply:
I think it'll be more intuitive. Change it if you can.

Reply:
Done